inc/analyzers.json

```diff
@@ -854,9 +854,7 @@
 				"kuromoji_baseform",
 				"ja_pos_filter",
 				"icu_normalizer",
-				"icu_folding",
-				"greek_lowercase_filter",
-				"cjk_width"
+				"icu_folding"
```

Comment:
Just to note here it turns out `icu_normalizer` actually does the job of `greek_lowercase_filter` and `cjk_width` already